server/rooms.ts

```diff
@@ -27,6 +27,7 @@ import {GTSGiveaway, LotteryGiveaway, QuestionGiveaway} from './chat-plugins/wif
 import {PM as RoomBattlePM, RoomBattle, RoomBattlePlayer, RoomBattleTimer} from "./room-battle";
 import {RoomGame, RoomGamePlayer} from './room-game';
 import {Roomlogs} from './roomlogs';
+import * as crypto from 'crypto';
```

Comment:
You forgot to remove the `require` from `core.js` (this is just an eslint warning so you may have overlooked it).